Cannot generate `edgeql-js` files with `@edgedb/generate@1.5.0`

**Describe the bug**
It seems the newest release of `@edgedb/generate@1.5.0` is broken because of `@iarna/toml`.

**Reproduction**
The edgeql-js generation command gives the following result:

```sh
$ npx @edgedb/generate edgeql-js --target=ts
node:internal/modules/cjs/loader:1152
  throw err;
  ^

Error: Cannot find module '@iarna/toml'
Require stack:
- /Users/XXXXXX/.npm/_npx/b55eec623338e3ca/node_modules/@edgedb/generate/dist/cli.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1149:15)
    at Module._load (node:internal/modules/cjs/loader:990:27)
    at Module.require (node:internal/modules/cjs/loader:1237:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/Users/XXXXXX/.npm/_npx/b55eec623338e3ca/node_modules/@edgedb/generate/dist/cli.js:28:27)
    at Module._compile (node:internal/modules/cjs/loader:1378:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1437:10)
    at Module.load (node:internal/modules/cjs/loader:1212:32)
    at Module._load (node:internal/modules/cjs/loader:1028:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/XXXXXX/.npm/_npx/b55eec623338e3ca/node_modules/@edgedb/generate/dist/cli.js'
  ]
}

Node.js v21.6.2
```

**Expected behavior**
The `dbschema/edgeql-js` should be generated w/o errors.

**Versions**

- OS: MacOS Sonoma 14.2.1
- EdgeDB version: `4.7+39afbed`
- EdgeDB CLI version: `4.1.1+e89e36b`
- `edgedb-js` version: `edgedb@1.5.0`
- Node/Deno version: `Node v21.6.2`

--- a/packages/generate/package.json
+++ b/packages/generate/package.json
@@ -13,13 +13,14 @@
   },
   "license": "Apache-2.0",
   "sideEffects": false,
-  "files": ["/dist"],
+  "files": [
+    "/dist"
+  ],
   "bin": "dist/cli.js",
   "peerDependencies": {
     "edgedb": "^1.5.0"
   },
   "devDependencies": {
-    "@iarna/toml": "^2.2.5",
     "@types/jest": "^29.5.2",
     "@types/node": "^20.3.2",
     "conditional-type-checks": "^1.0.6",
@@ -31,7 +32,9 @@
     "ts-jest": "^29.1.0",
     "typescript": "^5.4.3"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@iarna/toml": "^2.2.5"
+  },
   "scripts": {
     "typecheck": "tsc --project tsconfig.build.json --noEmit",
     "play": "tsx playground.ts",

